drivers/hmis/spec/factories/hmis/hud/exits.rb

```diff
@@ -11,6 +11,10 @@
     sequence(:ExitID, 50)
     destination { 1 }
     exit_date { Date.today }
+
+    trait :without_validations do
```

Comment:
This factory should already accept trait `skip_validate` that it inherits from parent `hmis_base_factory`